fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/AbstractArrayConstructorValue.java

```diff
@@ -83,11 +83,10 @@ public Type getResultType() {
 
     @Nonnull
     @Override
-    public String describe(@Nonnull final Formatter formatter) {
-        return "array(" + children.stream().map(child -> child.describe(formatter)).collect(Collectors.joining(", ")) + ")";
+    public String explain(@Nonnull final Formatter formatter) {
+        return "(" + children.stream().map(child -> child.explain(formatter)).collect(Collectors.joining(", ")) + ") as Array";
```

Comment:
why not just display it as a function? this almost looks like a cast.

Reply:
I thought it explains the intent better in a larger context, but I don't have a strong argument so I will switch to the old representation as suggested.

Reply:
maybe we just do `[...]` like a lot of other languages do

Reply:
done!